spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2641,4 +2641,20 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("test integral divide") {
```

Comment:
Would you mind adding more tests like using `makeParquetFileAllTypes`?

Reply:
Sure, I will add more tests later

Reply:
I have constructed the test dataset using `makeParquetFileAllTypes` and checked the div operations for all integral type and decimal type